features/contain-inline-size.yml

```
@@ -0,0 +1,6 @@
+name: Inline-size containment
+description: "The `contain: inline-size` CSS declaration TODO."
```

```suggestion
description: "The `contain: inline-size` CSS declaration prevents the element's inline size from being set by the element's contents. This permits the browser to avoid slower rendering calculations."
```

---

Do you think "layout calculations" would work just as well? To me, rendering is a larger process that also includes painting pixels.

---

Precision is good! Though I suspect for developers it's a case of "divs go brrr."

```suggestion
description: "The `contain: inline-size` CSS declaration prevents the element's inline size from being set by the element's contents. This permits the browser to avoid slower layout calculations."
```

---

Another take on this, to avoid the circular description:

```suggestion
description: "The `contain: inline-size` CSS declaration prevents the element's inline dimension from being set by the element's contents. This permits the browser to avoid slower layout calculations."
```